### PR TITLE
add branch Fuature/hw 5

### PR DESCRIPTION
**Pull request by YariLAN**

### CODE REVIEW

File: Student_School_DO/HW5/Responce/Responce.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Models
+{
+    public class Responce<T>
```
Comment (SolovyovDmitry) [inline Student_School_DO/HW5/Responce/Responce.cs]: typo

File: Student_School_DO/HW5/Responce/BookModel.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿namespace Models
+{
+    public class BookModel
```
Comment (SolovyovDmitry) [inline Student_School_DO/HW5/Responce/BookModel.cs]: more common name for dto(client-layer models) is BookInfo or BookResponse

**Review by SolovyovDmitry — APPROVED** [review]
Good for you to implement AutoMapper but that wasn't required)

File: Student_School_DO/HW5/Responce/Responce.cs (hunk 1 above)
Comment (YariLAN) [reply Student_School_DO/HW5/Responce/Responce.cs]: +

File: Student_School_DO/HW5/Responce/BookModel.cs (hunk 2 above)
Comment (YariLAN) [reply Student_School_DO/HW5/Responce/BookModel.cs]: +

**Pull request merged**